pydarn/plotting/superdarn_radars.py

```diff
@@ -1,101 +1,116 @@
-# Copyright (C) 2019 SuperDARN
-# Author: Marina Schmidt
+# Copyright (C) SuperDARN Canada, University of Saskatchewan
+# Authors: Marina Schmidt and Danno Peters
 
 """
 This module contains SuperDARN radar information
 """
 from typing import NamedTuple
 from enum import Enum
+from datetime import datetime
 
-
+"""
+_Hemispheres Class - Used to denote which hemisphere a radar is located in
+      _Hemisphere.North = 1
+      _Hemisphere.South = 0
```

Comment:
Ok, I'm going to suggest a change here and say that it should be North=1 and South=-1.  I feel very strongly about this, because it makes things easier when evaluating data; the sign of the hemisphere corresponds to the sign of the latitude.

Reply:
Done.